tesla_powerwall/const.py

```diff
@@ -20,10 +20,13 @@ class Roles(Enum):
 
 class GridStatus(Enum):
     CONNECTED = "SystemGridConnected"
-    ISLANEDED_READY = "SystemIslandedReady"
-    ISLANEDED = "SystemIslandedActive"
+    ISLANDED_READY = "SystemIslandedReady"
+    ISLANDED = "SystemIslandedActive"
```

Comment:
Note: these two lines are just fixing a typo in a (seemingly unused?) constant

```diff
-ISLANEDED
+ISLANDED
```

Reply:
The type is is used, because `get_grid_status` returns `GridStatus` which might have the `ISLANDED_READY` or `ISLANDED` status. But yea, thank you for changing the typo ;)